src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -46,6 +46,7 @@ public void SerializeHugeObjectGraphs(int limit)
         }
 
         [Theory]
+        [SkipOnCoreClrAttribute("Takes too long on Checked", RuntimeConfiguration.Checked)]
```

Comment:
Nit: remove the `Attribute` prefix.

Reply:
Thanks. That's the issue with copy-pasting from the actual attributes implementation ;)